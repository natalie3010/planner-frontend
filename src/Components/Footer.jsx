import React from 'react'
import { CG } from 'cap-shared-components'

export const Footer = () => {
  return (
    <CG.Footer style={{ padding: '20px' }}>
      <CG.HomeLink
        homeLink={{ name: 'workforce Planner', url: '/workforcePlanner' }}
        homeIcon={{ type: 'LogoAlt3', height: '26px', width: '26px' }}
      />
    </CG.Footer>
  )
}
