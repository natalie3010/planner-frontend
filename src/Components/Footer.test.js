import React from 'react'
import renderer from 'react-test-renderer'
import { Footer } from './Footer'

describe('Testing <Footer/> component', () => {
  it('tests snapshots', () => {
    const component = renderer.create(<Footer />).toJSON()
    expect(component).toMatchSnapshot()
  })
})
