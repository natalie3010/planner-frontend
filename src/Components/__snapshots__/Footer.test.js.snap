// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing <Footer/> component tests snapshots 1`] = `
<div
  className="sc-bYMpWt iaCdmq"
  style={
    {
      "marginTop": "auto",
    }
  }
>
  <div
    className="sc-hhOBVt gWMruW"
  >
    <span
      className="sc-ilhmMj igQPbE"
    >
      <svg
        alt="icon"
        height="26px"
        style={
          {
            "verticalAlign": "baseline",
          }
        }
        viewBox="0 0 558.23 493.84"
        width="26px"
        xmlns="http://www.w3.org/2000/svg"
      >
        <g
          fill="currentColor"
          stroke="currentColor"
        >
          <path
            className="cashorted-logo-alt-3_svg__a"
            d="M530.69 492.41c.21 1.06-.13 1.41-1.28 1.41q-21.9-.06-43.79 0H39.24a9 9 0 0 0-1.27 0c-1.16.19-1.5-.33-1.28-1.39.66-.16.62-.78.74-1.25.88-3.46 1.73-6.93 2.6-10.4a1.83 1.83 0 0 0 .28-.46c.72-2.66 1.48-5.32 2.12-8 .57-2.42 1.22-4.81 1.81-7.22 1-4.05 2-8.09 3-12.13.41-1.66 1.18-3.28 1-5.06 1.44-3.85 2.12-7.9 3.16-11.85a2.65 2.65 0 0 0 .25-.64c.41-3.07.4-3.07 3.44-3.07h457.4a10.76 10.76 0 0 0 1.27 0 1.23 1.23 0 0 1 1.51 1.09c.69 2.87 1.33 5.76 2.16 8.59.57 2 .68 4.06 1.7 5.89a4.49 4.49 0 0 0 .11 1.67c.45 1.62.89 3.24 1.3 4.87 1 4.09 2.08 8.17 3.09 12.26.7 2.82 1.24 5.68 2.06 8.46.55 1.87.68 3.86 1.67 5.6a5.84 5.84 0 0 0 .84 3.41c.59 2.34 1.16 4.69 1.76 7 .11.46.07 1.06.73 1.22Z"
          />
          <path
            className="cashorted-logo-alt-3_svg__b"
            d="M459.1 285.85c-.26 2.55-.26 2.55 2.17 2.55h27.2c1.09 0 1.44.23 1.4 1.37q-.14 4.32 0 8.64c0 1.24-.3 1.6-1.57 1.59h-28.05c-.41 0-1.07-.34-1.2.32a4.75 4.75 0 0 0 .05 2.09c.12.43.76.19 1.16.19h32.17c.6-.05.84.17.84.8v10.05c0 .55-.2.78-.75.74a8 8 0 0 0-.85 0h-48.31c-1.38 0-1.59-.44-1.58-1.66V274.3a3.23 3.23 0 0 1 1.54-.16h48.47a3.23 3.23 0 0 1 1.54.16v11.18a3.69 3.69 0 0 1-1.68.16h-31c-.56.03-1.1-.21-1.55.21ZM335.94 314.17h-17a1.87 1.87 0 0 1-.09-.41v-38.73c0-.82.35-.9 1-.89h28a54.92 54.92 0 0 1 12.77 1.52c4.41 1.14 8.54 2.92 11 7 3.6 5.82 2.35 14.62-4.21 18.57a30.62 30.62 0 0 1-7.51 3l13.46 9.61c-.26.49-.69.25-1 .25-5.86 0-11.71-.06-17.56.06a6.34 6.34 0 0 1-4.09-1.52c-3.05-2.23-6.1-4.46-9.16-6.66a1.87 1.87 0 0 0-1-.39 23.63 23.63 0 0 0-4.37.11c-.4.39-.17.88-.18 1.31-.01 2.43.22 4.81-.06 7.17Zm6.71-20.09c1.74 0 3.49.06 5.23 0 2.35-.11 4.72-.26 6.82-1.46a3 3 0 0 0 .24-5.29 8 8 0 0 0-3.28-1.18c-4.91-.8-9.86-.33-14.8-.47-.79 0-.78.45-.78 1v6.5c0 .76.24 1 1 .95 1.86-.08 3.72-.05 5.6-.05ZM155.94 314.15h-9.24c-.45-.32-1-.14-1.44-.18a46.73 46.73 0 0 1-12.43-2.12 13.71 13.71 0 0 1-7.69-5.56c-.9-1.37-1-3-1.42-4.47-.21-.73.38-.68.86-.68H137.32c1.6 0 3.19-.27 4.74.73a6.58 6.58 0 0 0 3.26.58 104.44 104.44 0 0 0 12-.09 8.6 8.6 0 0 0 4.44-1.2c-4.55-1.71-9-1.14-13.32-1.21-.66-.35-1.38-.08-2.05-.19a47.48 47.48 0 0 1-14-2.35 14.55 14.55 0 0 1-6.49-4.27 8.44 8.44 0 0 1-2-5.36v-1.68c.35-5.39 4-8 8.51-9.58a42.23 42.23 0 0 1 10.9-2.08c2.06-.18 4.12-.14 6.17-.31h3.68a58.59 58.59 0 0 1 14.68 1.78c3.49 1 6.79 2.31 9 5.4a9.89 9.89 0 0 1 1.79 5.6 10.88 10.88 0 0 1-1.12.11h-16.13c-1.28-1.22-3-.89-4.52-1.19a14.61 14.61 0 0 0-2.81 0 18.44 18.44 0 0 0-5.55 0 23.46 23.46 0 0 0-6.36.52c-.47.11-1 .21-1 .81s.64.51 1 .61a10.32 10.32 0 0 0 3.87.37c.55.34 1.15.12 1.73.19 2.26 0 4.53 0 6.79.11 5.31.23 10.61.63 15.65 2.54 3.49 1.31 6.58 3.17 7.88 6.95 1.78 5.18-.41 10-5.74 12.72a30.84 30.84 0 0 1-10.34 2.91c-1.53.18-3.07.28-4.6.42-.44.06-1-.14-1.44.17ZM532.13 274.14a49.46 49.46 0 0 1 11.62 1.53c5.07 1.42 9.47 3.81 12.18 8.6a17.74 17.74 0 0 1 2.19 7.44c.39 4.83-.19 9.49-3 13.63a18 18 0 0 1-8.87 6.52 33.11 33.11 0 0 1-10 2c-10.88.59-21.76.09-32.64.27a8 8 0 0 1-.11-1v-38c0-1 .42-1.1 1.24-1.1q13.64.13 27.39.11Zm-11.52 20v7.5c0 .73.18 1 .94 1 3.64 0 7.28.1 10.9-.07 3.09-.14 6-1 7.61-4 3.14-5.92-.38-12.21-7-12.7-3.77-.28-7.54-.05-11.31-.17-.9 0-1.12.28-1.1 1.13 0 2.39-.04 4.85-.04 7.3ZM226.28 300.08a12.92 12.92 0 0 0-3.23-.12h-15.72c-1 0-1.28.2-1.26 1.22.06 3.87 0 7.74.05 11.61 0 1.06-.19 1.45-1.37 1.44-4.87-.08-9.73-.07-14.59 0-1.06 0-1.32-.24-1.32-1.31q.06-19.32 0-38.65l16.73-.07a2.18 2.18 0 0 1 .54.15v13.17c0 .45-.14.94.64.93 6.52-.05 13-.07 19.54-.1.41-.39.19-.89.19-1.32v-11.71c0-.51-.25-1.15.73-1.14 5.45.05 10.91.06 16.36.09v38.5c0 1.14-.23 1.48-1.44 1.47-5.19-.08-10.39 0-15.58 0-.25-4.2 0-8.42-.12-12.62.03-.56.27-1.1-.15-1.54ZM282.69 274.15a46.62 46.62 0 0 1 12.59 1.82c4.53 1.43 8.41 3.8 10.9 8a19 19 0 0 1 2.38 7.82c.08.48-.14 1 .2 1.46-.23 3.2-.38 6.41-1.73 9.4a17.07 17.07 0 0 1-10.4 9.18 39.51 39.51 0 0 1-10.59 2.11 68 68 0 0 1-9.13.14c-5.58-.38-11.1-1.35-15.87-4.54-4.16-2.79-6.29-6.9-7-11.82a23 23 0 0 1 1.13-11.61 16.35 16.35 0 0 1 9.34-9.28c4.87-2.06 10.07-2.56 15.31-2.7Zm-1.41 28.42h1.56c5.22-.25 8.16-2.75 8.56-7.69.37-4.54-1.69-7.81-6.26-8.79a18.45 18.45 0 0 0-6.58-.23c-3.24.48-6 1.7-7 5.2s-.82 6.78 2.05 9.35c2.14 1.92 4.9 2.2 7.67 2.16ZM99.83 274.33c1.54 3.63 3.53 7.05 5.33 10.55q7.13 13.89 14.26 27.75c.79 1.55.81 1.54-.87 1.54h-15.3c-1 0-1.6-.15-1.93-1.14a2.38 2.38 0 0 0-2.69-1.75c-6.38.1-12.76.07-19.13 0-1.06 0-1.66.28-2 1.25-.4 1.27-1.16 1.69-2.57 1.66-5.43-.11-10.86-.05-16.58-.05 6.9-13.4 13.71-26.65 20.52-39.89a7.79 7.79 0 0 1 1-.12h18.86a6.8 6.8 0 0 1 1.1.2Zm-5.33 25.39-5.18-10c-1.78 3.46-3.44 6.69-5.14 10ZM26.37 285.82c-3.18.07-6.23.65-7.92 3.67-2.32 4.16-1.72 11.74 5.63 12.81 3.93.57 7.79.59 11.42-1.41a3.65 3.65 0 0 0 2-3c.07-.94.35-1 1.13-1h16.15l.1 1.4c-.51 6.76-4.31 10.94-10.43 13.31a38 38 0 0 1-11.51 2.36c-.49.06-1-.09-1.45.21-2.69 0-5.39.11-8.07-.08a37.79 37.79 0 0 1-11.48-2.24 21 21 0 0 1-5.34-2.87 15 15 0 0 1-5.54-7.87c-1.77-6.37-1.62-12.59 2.15-18.28a16.48 16.48 0 0 1 7.78-6c4.87-2 10-2.43 15.14-2.66h2.82a54.67 54.67 0 0 1 11.12 1.24c5 1.2 9.7 3 12.63 7.7a14.93 14.93 0 0 1 2.11 6.73c.29 1.27-.24 1.55-1.48 1.53-4.81-.07-9.62 0-14.42 0-.59 0-1.26.15-1.43-.7-.23-2.25-1.84-3.28-3.68-4a10.38 10.38 0 0 0-4.81-.76 4.32 4.32 0 0 0-2.62-.09ZM380.21 285.55v-11.29a2.1 2.1 0 0 1 .42-.09h50.09c.81 0 .9.33.9 1 0 3.11-.06 6.22 0 9.33 0 1-.32 1.23-1.27 1.22-4.81-.05-9.62 0-14.43 0-.44 0-.94-.23-1.33.18 0 .52-.09 1-.09 1.54v25.31c0 1.32-.4 1.51-1.57 1.49-4.71-.07-9.43-.09-14.14 0-1.37 0-1.59-.43-1.59-1.65.05-8.39 0-16.78 0-25.17a3.18 3.18 0 0 0-.12-1.53c-1.32-.45-2.67-.15-4-.18-4.24-.15-8.56.09-12.87-.16Z"
          />
          <path
            className="cashorted-logo-alt-3_svg__a"
            d="M283.55 0h.28c0 .48.37.6.69.8 3.09 2 6.16 3.92 9.26 5.84s6.39 3.9 9.59 5.84l13.33 8.22a37.3 37.3 0 0 0 4.67 3.08c2.76 1.76 5.58 3.42 8.37 5.13l13.93 8.75a4.71 4.71 0 0 0 .42.36l14 8.75 9.64 6q5.25 3.25 10.49 6.52a75.71 75.71 0 0 0 7.33 4.54 38.63 38.63 0 0 0 4.85 3.14l23.54 14.63a4.19 4.19 0 0 0 1.41.9 23.82 23.82 0 0 0 4.21 2.57 2.16 2.16 0 0 0 .29.31l15.05 9.27 2.25 1.47 4.57 2.77 3.7 2.25c2.28 1.47 4.54 3 6.86 4.4a50 50 0 0 0 5.53 3.32 5.26 5.26 0 0 0 2.36 1.34 3.7 3.7 0 0 0 1.68 1.16 1.71 1.71 0 0 0 .36.43l12.07 7.54a77.17 77.17 0 0 0 6.93 4.13.22.22 0 0 1 .19.06 40.13 40.13 0 0 0 6.15 4c2.59 1.81 5.32 3.41 8 5l7.85 4.87c2.54 1.89 5.35 3.37 8 5.05l1.79 1.11c4.66 2.93 9.33 5.84 14 8.79.61.38 1.81.67 1.63 1.29-.25.88-1.4.35-2.13.35H40.63c-.74 0-1.94.49-2.15-.32s1.09-.94 1.73-1.38c2.71-1.84 5.69-3.28 8.27-5.34l15-9.36a104.51 104.51 0 0 0 10.48-6.5c4.66-2.88 9.39-5.67 13.94-8.74a3.19 3.19 0 0 0 1.7-1.13l5.68-3.66a11.56 11.56 0 0 0 2.5-1.51 33.14 33.14 0 0 0 6.28-3.91l3.1-2q4.38-2.46 8.53-5.31l8.47-5.19a1.88 1.88 0 0 0 .53-.14l11.46-7.16c1.84-1.11 3.68-2.2 5.5-3.33 2.54-1.59 5.14-3.08 7.55-4.87 2.91-1.82 5.83-3.63 8.72-5.48.73-.47 1.67-.69 2.06-1.6.21-.07.48 0 .58-.3a1.89 1.89 0 0 0 .69-.1c2.92-1.8 5.9-3.49 8.66-5.54l5.91-3.74a12 12 0 0 0 3.7-2.25l8.2-5.15c.33-.17.67-.31 1-.51l14.91-9.34a59.87 59.87 0 0 0 6.88-4.23l6.45-4c.84-.52 1.69-1 2.51-1.56 2.76-1.81 5.72-3.34 8.28-5.48l5.94-3.7 5.34-3.18a20.62 20.62 0 0 0 4.85-3.07l.52-.35a4.1 4.1 0 0 0 2-1.09 25.52 25.52 0 0 0 3.54-2.13c1.26-1 2.93-1.45 4.11-2.63l8.52-5.37c3.37-1.73 6.45-3.92 9.64-5.94s6.51-3.92 9.7-6c.57-.36 1.48-.48 1.64-1.39Z"
          />
          <path
            className="cashorted-logo-alt-3_svg__b"
            d="M315.03 185.58h59.34c1.13 0 1.74.19 1.33 1.49a4.13 4.13 0 0 0 0 1.27v65.81a3.37 3.37 0 0 1-1.93.28h-58.48v-64.64a16.52 16.52 0 0 0-.27-4.21ZM252.36 185.57v67.39c0 2.86 0 2.86-2.89 2.86h-55.81c-1.95 0-2 0-2-2v-65.49a3.86 3.86 0 0 0 0-1.13c-.43-1.4.19-1.64 1.46-1.64h59.22ZM67.87 185.86a2.1 2.1 0 0 1 1.34-.3h59.54v68.24c0 1.9-.13 2-2 2H70.21c-2.07 0-2.1 0-2.1-2.18V187.8a3.91 3.91 0 0 0-.24-1.94ZM438.63 185.59h59.53a2.11 2.11 0 0 1 1.35.3 3.91 3.91 0 0 0-.25 1.94v65.42c0 2.6 0 2.6-2.52 2.6h-56.1c-1.91 0-2-.09-2-2q-.03-34.14-.01-68.26ZM191.7 400.59V338c0-2.2.08-2.29 2.34-2.29h56.25c1.78 0 1.78 0 1.78 1.75v61.46c0 1.4-.42 1.7-1.75 1.7q-29.33-.06-58.66 0ZM375.68 400.59h-58.65c-1.73 0-1.74 0-1.74-1.79v-61.31c0-1.77 0-1.78 1.75-1.78h56.66c1.89 0 2 .08 2 2v62.87ZM128.49 367.69v31.02c.13 1.88.13 1.88-1.73 1.88H70.09c-2 0-2 0-2-1.94v-60.47a9.8 9.8 0 0 1 .12-1.54c.11-.62.42-1 1.15-.93a5.61 5.61 0 0 0 .71 0h56.68c1.34 0 1.75.32 1.74 1.71-.02 10.07 0 20.17 0 30.27ZM499.26 368.33v30.31c0 1.92 0 1.93-1.84 1.93h-57c-1.75 0-1.74 0-1.58-1.75v-61.18c0-1.95 0-1.95 2-1.95h56.11c2.18 0 2.26.08 2.26 2.34Z"
          />
        </g>
      </svg>
    </span>
    <a
      className="sc-jIRcFI ksqgpu"
      href="/workforcePlanner"
    >
      workforce Planner
    </a>
  </div>
</div>
`;
